.github/workflows/post-release.yml

```diff
@@ -42,3 +42,49 @@ jobs:
           context: .
           push: true
           tags: ${{ env.IMAGE_NAME }}:latest,${{ env.IMAGE_NAME }}:${{ steps.tag.outputs.tag }}
+
+  helm:
+
+    needs: docker
+
+    name: Helm Deploy Dev
+    runs-on: ubuntu-latest
+
+    env:
+      CLUSTER_RESOURCE_GROUP_NAME: caf-aks-lz-dev-rg
+      CLUSTER_NAME: aks-lz-dev
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: AZ Login
+        env:
+          SERVICE_PRINCIPAL_APP_ID: ${{ secrets.CLUSTER_CRED_GETTER_APPID }}
+          SERVICE_PRINCIPAL_SECRET: ${{ secrets.CLUSTER_CRED_GETTER_PWD }}
+          SERVICE_PRINCIPAL_TENANT: ${{ secrets.CLUSTER_TENANTID }}
+        run: |
+          az login \
```

Comment:
maybe we should use this instead?  https://github.com/Azure/login

Reply:
converted to use this action